packages/atomic/src/components/insight/atomic-insight-full-search-button/atomic-insight-full-search-button.pcss

```diff
@@ -0,0 +1,28 @@
+@import '../../../global/global.pcss';
+
+:host {
+  &::part(full-search-button) {
```

Comment:
Wouldn't it be possible to use tailwind classes for a lot of those ?

There's also hardcoded colors which should be variables.